### PR TITLE
tests: add mtxlora commands

### PR DESCRIPTION
**Pull request by nickkbell**

### CODE REVIEW

File: tests/fixtures/analog/commands.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ export const uplinkCommands = {
     'new event': {
         hex: {
             dump: '1506081e000973cbe1',
-            lrc: 0xe1
+            lrc: '0xe1'
```
Comment (DarkPark) [inline tests/fixtures/analog/commands.js:5]:
```suggestion
            lrc: 0xe1
```

File: tests/specs/mtxLora/downlinkCommands.spec.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import {downlinkCommands} from '../../fixtures/mtxLora/commands.js';
+import {validateMtxLoraMessage} from './utils.js';
+
+
+test.describe('mtxLora downlink commands - create messages', () => {
+    test.beforeEach(async ( {page, baseURL} ) => {
+        await page.goto(baseURL);
+        await new MainPage(page).selectCodec(fixture.codecType.options.MTX_LORA);
+    });
+    test.afterEach(async ({page}) => page.getByLabel(fixture.logs.buttons.deleteLogs).click());
```
Comment (DarkPark) [inline tests/specs/mtxLora/downlinkCommands.spec.js:13]:
```suggestion

    test.afterEach(async ({page}) => page.getByLabel(fixture.logs.buttons.deleteLogs).click());
```

File: tests/fixtures/analog/commands.js (hunk 1 above)
Comment (nickkbell) [reply tests/fixtures/analog/commands.js:5]: no, assertion as a text

File: tests/specs/mtxLora/uplinkCommands.spec.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import {validateMtxLoraMessage} from './utils.js';
+
+
+test.describe('mtxLora uplink commands - parse hex dumps', () => {
+    test.beforeEach(async ( {page, baseURL} ) => {
+        await page.goto(baseURL);
+        await new MainPage(page).selectCodec(fixture.codecType.options.MTX_LORA);
+        await page.getByLabel(fixture.parseMessages.directions.uplink).click();
+    });
+    test.afterEach(async ({page}) => page.getByLabel(fixture.logs.buttons.deleteLogs).click());
```
Comment (DarkPark) [inline tests/specs/mtxLora/uplinkCommands.spec.js:14]:
```suggestion
    test.afterEach(async ( {page} ) => page.getByLabel(fixture.logs.buttons.deleteLogs).click());
```

and in other places

File: tests/specs/mtxLora/uplinkCommands.spec.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+    for ( const [commandKey, command] of Object.entries(uplinkCommands) ) {
+        test(`check ${commandKey}`, async ({page}) => {
+            const mainPage = await new MainPage(page);
+
+            await mainPage.parseDump(command.base64);
+
```
Comment (DarkPark) [inline tests/specs/mtxLora/uplinkCommands.spec.js:41]:
```suggestion
```

File: tests/fixtures/analog/commands.js (hunk 1 above)
Comment (DarkPark) [reply tests/fixtures/analog/commands.js:5]: why not a number?

Comment (nickkbell) [reply tests/fixtures/analog/commands.js:5]: because in UI  lrc  is a text and not a number. also playwright method getByText() - expects a text

**Review by aleks-and — APPROVED** [review]

**Review by DeadByElpy — APPROVED** [review]

File: .eslintrc.cjs [hunk 5]
```diff
@@ -128,6 +128,15 @@ module.exports = {
         'no-await-in-loop': 'off',
         'no-continue': 'off',
         'no-bitwise': 'off',
-        'no-restricted-syntax': 'off'
+        'no-restricted-syntax': 'off',
+
+        //tests
```
[the remark this answers is not in the document]
Comment (nickkbell) [reply .eslintrc.cjs]: fixed

File: tests/specs/mtxLora/downlinkCommands.spec.js (hunk 2 above)
Comment (nickkbell) [reply tests/specs/mtxLora/downlinkCommands.spec.js]: fixed

File: tests/specs/mtxLora/uplinkCommands.spec.js (hunk 3 above)
Comment (nickkbell) [reply tests/specs/mtxLora/uplinkCommands.spec.js]: fixed

File: tests/specs/mtxLora/uplinkCommands.spec.js (hunk 4 above)
Comment (nickkbell) [reply tests/specs/mtxLora/uplinkCommands.spec.js]: fixed

File: tests/specs/mtxLora/downlinkCommands.spec.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+test.describe('mtxLora downlink commands - create messages', () => {
+    test.beforeEach(async ( {page, baseURL} ) => {
+        await page.goto(baseURL);
+        await new MainPage(page).selectCodec(fixture.codecType.options.MTX_LORA);
+    });
+
+    test.afterEach(async ( {page} ) => page.getByLabel(fixture.logs.buttons.deleteLogs).click());
+
+    for ( const [commandKey, command] of Object.entries(downlinkCommands) ) {
+        test(`check ${commandKey}`, async ({page}) => {
```
Comment (DarkPark) [inline tests/specs/mtxLora/downlinkCommands.spec.js:17]: https://github.com/jooby-dev/jooby-gui/pull/36#discussion_r1682968730

File: tests/specs/mtxLora/uplinkCommands.spec.js [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+    test.beforeEach(async ( {page, baseURL} ) => {
+        await page.goto(baseURL);
+        await new MainPage(page).selectCodec(fixture.codecType.options.MTX_LORA);
+        await page.getByLabel(fixture.parseMessages.directions.uplink).click();
+    });
+
+    test.afterEach(async ( {page} ) => page.getByLabel(fixture.logs.buttons.deleteLogs).click());
+
+    for ( const [commandKey, command] of Object.entries(uplinkCommands) ) {
+        test(`check ${commandKey}`, async ({page}) => {
```
Comment (DarkPark) [inline tests/specs/mtxLora/uplinkCommands.spec.js:18]: https://github.com/jooby-dev/jooby-gui/pull/36#discussion_r1682968730

File: tests/specs/mtxLora/uplinkCommands.spec.js [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+        await page.goto(baseURL);
+        await new MainPage(page).selectCodec(fixture.codecType.options.MTX_LORA);
+        await page.getByLabel(fixture.parseMessages.format.base64).click();
+        await page.getByLabel(fixture.parseMessages.directions.uplink).click();
+    });
+
+    test.afterEach(async ( {page} ) => page.getByLabel(fixture.logs.buttons.deleteLogs).click());
+
+    for ( const [commandKey, command] of Object.entries(uplinkCommands) ) {
+        test(`check ${commandKey}`, async ({page}) => {
```
Comment (DarkPark) [inline tests/specs/mtxLora/uplinkCommands.spec.js:39]: https://github.com/jooby-dev/jooby-gui/pull/36#discussion_r1682968730

File: tests/specs/mtxLora/uplinkCommands.spec.js [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+            const mainPage = await new MainPage(page);
+
+            await mainPage.parseDump(command.hex.dump);
+
```
Comment (DarkPark) [inline tests/specs/mtxLora/uplinkCommands.spec.js:22]:
```suggestion
```

File: tests/specs/mtxLora/downlinkCommands.spec.js [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+            if ( command.hardwareType ) {
+                await mainPage.selectHardwareType(command.hardwareType);
+            }
+
+            for ( const [, subCommand] of Object.entries(command.mtx.commands) ) {
+                await mainPage.createMessage(subCommand, 'downlink');
+            }
+
+            await mainPage.buildMessage();
+
```
Comment (DarkPark) [inline tests/specs/mtxLora/downlinkCommands.spec.js:29]:
```suggestion
```

File: tests/specs/mtxLora/downlinkCommands.spec.js (hunk 6 above)
Comment (nickkbell) [reply tests/specs/mtxLora/downlinkCommands.spec.js]: fixed

File: tests/specs/mtxLora/uplinkCommands.spec.js (hunk 7 above)
Comment (nickkbell) [reply tests/specs/mtxLora/uplinkCommands.spec.js]: fixed

File: tests/specs/mtxLora/uplinkCommands.spec.js (hunk 8 above)
Comment (nickkbell) [reply tests/specs/mtxLora/uplinkCommands.spec.js]: fixed

File: tests/specs/mtxLora/uplinkCommands.spec.js (hunk 9 above)
Comment (nickkbell) [reply tests/specs/mtxLora/uplinkCommands.spec.js]: fixed

File: tests/specs/mtxLora/downlinkCommands.spec.js (hunk 10 above)
Comment (nickkbell) [reply tests/specs/mtxLora/downlinkCommands.spec.js]: fixed

**Pull request merged**